pet-store-react-hooks/src/PetModal/PetModal.tsx

```diff
@@ -256,6 +256,10 @@ export function PetModal(props: PetModalProps) {
           <button
             className="modal-close-header-btn"
             type="button"
+            // Question: This is a button. Should be labeled as such.
+            // I've read https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
+            // but not quite get what i'm supposed to label it as
+            aria-label="close modal"
```

Comment:
In our case it plays the role of a close button. `Close modal` should be good enough.